api/src/main/resources/workbench.yaml

```diff
@@ -684,6 +710,36 @@ definitions:
         type: integer
         format: int64
         description: Milliseconds since the UNIX epoch.
+      userRoles:
+        type: "array"
+        items:
+           $ref: "#/definitions/UserRole"
+
+  UserRole:
+    type: object
+    required:
+      - user
+      - role
+    properties:
+      user:
+        type: string
+      role:
+        $ref: "#/definitions/WorkspaceAccessLevel"
+
+  UserRoleList:
+    type: object
+    required:
+     - items
+    properties:
+      items:
+        type: "array"
+        items:
+           $ref: "#/definitions/UserRole"
+
+  WorkspaceAccessLevel:
+    type: string
+    description: levels of access to workspace
+    enum: &WORKSPACE_ACCESS_LEVEL [reader, writer, owner]
```

Comment:
OK to omit `&WORKSPACE_ACCESS_LEVEL` since we aren't referencing the value elsewhere.